js/chat.js

```diff
@@ -1,4 +1,5 @@
 var request;
+var api_root = "https://www.hackmud.com"
```

Comment:
I know you didn't write the code around here, but can we change this var to let?